bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/handlers/AbstractSolaxHandler.java

```diff
@@ -116,6 +122,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
+    @Override
+    protected void updateStatus(ThingStatus status, ThingStatusDetail statusDetail, @Nullable String description) {
+        super.updateStatus(status, statusDetail, description);
+        if (status == ThingStatus.OFFLINE && statusDetail == ThingStatusDetail.CONFIGURATION_ERROR) {
```

Comment:
```suggestion
        if (ThingStatus.OFFLINE.equals(status) && ThingStatusDetail.CONFIGURATION_ERROR.equals(statusDetail)) {
```

Reply:
Why do you prefer the usage of equals instead of == in the enum case?